controllers/update.go

```diff
@@ -500,7 +500,7 @@ func getComponentName(gitSource *appstudiov1alpha1.GitSource) string {
 	repoName := strings.Split(lastElement, ".git")[0]
 	componentName := repoName
 	context := gitSource.Context
-	if context != "" && context != "./" {
+	if context != "" && context != "./" && context != "." {
```

Comment:
how about `/`

Reply:
I'm wondering is `/` is a valid context.. what does root dir means within a given git repo?